tapgarden/planter.go

```diff
@@ -566,6 +566,17 @@ func (c *ChainPlanter) fundGenesisPsbt(ctx context.Context,
 			return nil, fmt.Errorf("unable to estimate fee: %w",
 				err)
 		}
+		minRelayFee, err := c.cfg.Wallet.MinRelayFee(ctx)
+		if err != nil {
+			return nil, fmt.Errorf("unable to obtain minimum relay"+
```

Comment:
nit: for ease of review, we standardize putting the space between words on the previous line if we have to wrap a string.